content/sensu-go/5.0/reference/mutators.md

```diff
@@ -169,6 +169,22 @@ example      | {{< highlight shell >}} "annotations": {
 
 ## Examples
 
+### Minimum required mutator attributes
+
+{{< highlight json >}}
+{
+  "type": "Mutator",
+  "api_version": "core/v2",
+  "metadata": {
+    "name": "mutator_minimum",
+    "namespace": "default"
+  },
+  "spec": {
+    "command": "example_mutator.go"
+  }
+}
+{{< /highlight >}}
+
```

Comment:
Can we move this example below the other example? It will flow better considering the text in lines 188-189.

Reply:
Yup! Crossed my mind when I added it that the text currently there made a weird break up. Done!